docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -35,70 +19,36 @@ All occurrences of the code before the `>>` must be substituted with the code af
 - `="conditionType" count="XOR"` >> `="conditionType" count="1"`
 - `="conditionType" count="-reset-XOR"` >> `="conditionType" count="-reset-1"`
 - `="isTwoHit" count="true"` >> `="hitPoints" count="2"`
-- `="isTwoHit" value="false"` >> `="hitPoints" value="1"`
-- `="isTwoHit" count="-reset-true"` >> `="hitPoints" count="-reset-2"`
 - `="isTwoHit" count="false"` >> `="hitPoints" count="1"`
+- `="isTwoHit" value="true"` >> `="hitPoints" value="2"`
```

Comment:
I would have this and the following line before the other "isTwoHit" related lines, since this line and the following one are the two most important ones (by far). Practically this suggestion would change this part of the listing to this order:
- `="isTwoHit" value="true"` >> `="hitPoints" value="2"`
- `="isTwoHit" value="false"` >> `="hitPoints" count="1"`
- `="isTwoHit" count="true"` >> `="hitPoints" count="2"`
- `="isTwoHit" count="false"` >> `="hitPoints" count="1"`
- `="isTwoHit" count="-reset-true"` >> `="hitPoints" count="-reset-2"`
- `="isTwoHit" count="-reset-false"` >> `="hitPoints" count="-reset-1"`

Reply:
I suppose it's a bit arbitrary how to order the list. I appreciate the suggestion though for the ordering to make more sense.

I pushed an update to alpha-sort the list: 81b5d1c48

Since it is a bit arbitrary, that at least gives a consistent ordering and keeps related elements close together.

Reply:
I guess that's acceptable but, sadly, this means that the much less important "value" ones come after the "count" ones. I guess this is a minor matter.